zulipterminal/model.py

```diff
@@ -1012,12 +1021,18 @@ def get_all_users(self) -> List[Dict[str, Any]]:
             for properties in self.user_dict.values()
             if properties["status"] == "inactive"
         ]
+        bot = [
+            properties
+            for properties in self.user_dict.values()
+            if properties["status"] == "bot"
+        ]
```

Comment:
This is different behavior than what you had before, and what we have now, so it at least warrants mentioning what changed in the commit text. I'd almost suggest this could be a separate commit, since it's a different change - the ordering, not the icon.

Reply:
Rebased anda added a new commit.